src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -546,8 +545,35 @@ def _initialize(self) -> None:
         self._engine = create_engine(
             url=url, connect_args=connect_args, **engine_args
         )
+
+        self.migrate_database()
         SQLModel.metadata.create_all(self._engine)
 
+    def migrate_database(
+            self,
+            revision="head"
+    ):
+        """ Migrate the Database to the head as defined by the current python package
+
+        Args:
+            revision: Version of the db-schema to up/downgrade to
+        """
+        from alembic.config import Config
+        from alembic import command
+
+        url, connect_args, engine_args = self.config.get_sqlmodel_config()
+        connection = self._engine.connect()
+
+        # TODO: This feels wrong, any suggestions how to improve this
+        ini_file_path = Path.absolute(Path(__file__).parents[1] / "alembic.ini")
```

Comment:
Would really like to find a better way to do this

Reply:
`Path.absolute(Path(zenml.__file__).parents[0] / "alembic.ini")` 😛

Reply:
Potentially a tiny bit better, as we can now freely move the `sql_zen_store.py` without messing this up

Reply:
No longer necessary as alembic won't be packaged

Reply:
Even better: we don't involve the `alembic.ini` file at all in the zenml db auto-migration.